<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGSBOWnd</class>
 <widget class="QWidget" name="CGSBOWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>过滤</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>筛选：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_pBOInfoComboBox">
        <item>
         <property name="text">
          <string>显示全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示单点通道</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示双点通道</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示浮点值通道</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示整型值通道</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>名称包含：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pBOInfoLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>428</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,1,5">
     <item>
      <widget class="QTableWidget" name="m_pGSBOTableWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>来源表</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>合位关联通道</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>分位关联通道</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_pCnnPushButton">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pDisPushButton">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>来源表：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_pCnnInfoComboBox">
           <item>
            <property name="text">
             <string>跳闸矩阵表</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>开关量表</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>模拟量表</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>名称包含：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_pCnnInfoLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="m_pCnnInfoTableWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>描述</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
